File: AndroidSample/src/main/java/com/example/activities/MainActivity.java
package com.example.activities;

import android.app.Activity;

import com.example.R;

import org.androidannotations.annotations.Bean;
import org.androidannotations.annotations.EActivity;

@EActivity(R.layout.activity_main)
public class MainActivity extends Activity {

    @Bean
    MainPresenter presenter;
}

File: app/src/main/java/com/template/project/MainActivity.kt
package com.template.project

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.findNavController
import org.androidannotations.annotations.EActivity

@SuppressLint("Registered")
@EActivity(R.layout.activity_main)
class MainActivity : AppCompatActivity() {

    override fun onSupportNavigateUp() = findNavController(R.id.nav_host_fragment).navigateUp()
}
